perception_launch/launch/object_recognition/detection/camera_lidar_fusion_based_detection.launch.xml

```diff
@@ -20,43 +20,43 @@
   <arg name="image_number" default="1"/>
   <arg name="use_vector_map" default="true"/>
 
-  <!-- <include file="$(find euclidean_cluster)/launch/euclidean_cluster.launch"/> -->
+  <!-- <include file="$(find-pkg-share euclidean_cluster)/launch/euclidean_cluster.launch.xml"/> -->
   <!-- or -->
-  <!-- <include file="$(find euclidean_cluster)/launch/voxel_grid_based_euclidean_cluster.launch" /> -->
+  <!-- <include file="$(find-pkg-share euclidean_cluster)/launch/voxel_grid_based_euclidean_cluster.launch.xml" /> -->
   <!-- or -->
-  <include file="$(find lidar_apollo_instance_segmentation)/launch/lidar_apollo_instance_segmentation.launch">
-    <arg name="output/objects" default="clusters"/>
+  <include file="$(find-pkg-share lidar_apollo_instance_segmentation)/launch/lidar_apollo_instance_segmentation.launch.xml">
+    <let var="output/objects" value="clusters"/>
   </include>
 
   <!-- Jetson AGX -->
-  <!-- <include file="$(find tensorrt_yolo3)/launch/yolo3.launch">
-    <arg name="image_raw0" default="$(arg image_raw0)"/>
-    <arg name="image_raw1" default="$(arg image_raw1)"/>
-    <arg name="image_raw2" default="$(arg image_raw2)"/>
-    <arg name="image_raw3" default="$(arg image_raw3)"/>
-    <arg name="image_raw4" default="$(arg image_raw4)"/>
-    <arg name="image_raw5" default="$(arg image_raw5)"/>
-    <arg name="image_raw6" default="$(arg image_raw6)"/>
-    <arg name="image_raw7" default="$(arg image_raw7)"/>
-    <arg name="image_number" default="$(arg image_number)"/>
+  <!-- <include file="$(find-pkg-share tensorrt_yolo3)/launch/yolo3.launch.xml">
+    <let var="image_raw0" value="$(var image_raw0)"/>
```

Comment:
Sorry, I think I was the one to change arg to `let`, but I later found out that setting variables using `let` wouldn't pass them to included file.
Could you switch it back to `arg`?
You have to use arg with `value` as shown in the suggestion. (This sounds weird since the [migration guide](https://index.ros.org/doc/ros2/Tutorials/Launch-files-migration-guide/#arg) states that `value` for arg is not supported anymore, but that was the only way to pass arguments to the included launch file)
 ```suggestion
    <arg var="image_raw0" value="$(var image_raw0)"/>
```

Reply:
Done in 7ae696e